src/browser/deferred.ts

```diff
@@ -32,7 +33,7 @@ interface Bounds {
 }
 
 function handleMove(windowId: string, bounds: Bounds): void {
-    const browserWindow = BrowserWindow.fromId(windowId);
+    const browserWindow = <BrowserWindowOF>BrowserWindow.fromId(windowId);
```

Comment:
perhaps `BrowserWindow.fromId(windowId) as BrowserWindowOF` ... this is considered the newer (better?) way of doing it. I cannot find what I read a long time ago to justify this, but perhaps we should agree on one style or the other.

Reply:
Ok, why not, will change 👍

Reply:
Nope, tslint gave an error, Microsoft thinks otherwise - check `prefer-type-cast` [here](https://github.com/Microsoft/tslint-microsoft-contrib)